Assets/MixedRealityToolkit.Examples/Experimental/HandMenuLayout/Prefabs/HandMenu_Slider_Assets/Scripts/UpdateTrackScale.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
```

Comment:
Nit: We haven't been using underscores in folder names (i.e. HandMenu_Slider_Assets -> HandeMenuSliderAssets)

Reply:
Thanks! Updating now.